code/demo/android-native/app/src/main/Application.mk

```diff
@@ -1,4 +1,5 @@
 APP_ABI := armeabi-v7a x86 arm64-v8a
```

Comment:
APP_ABI should also compile for x86_64.
It's mandatory if the app will be published on Google Play Store

Reply:
I got `Android NDK: android-14 is unsupported. Using minimum supported version android-16.` when building with the latest NDK. So I guess it now has to be minimum 16.

Reply:
indeed

Reply:
That'll be post-merge of this PR though, as I'll need to build all the dependencies in `orx-extern` for `x86_64` as well.